website/docs/guides/version-4-upgrade.html.md

```diff
@@ -99,6 +99,7 @@ provider "aws" {
 
 Version 4.0.0 adds these new provider arguments:
 
+* `assume_role.0.duration` - The assume role duration represented as a string e.g. `"1h"` or `"1h30s"`. Replaces `assume_role.0.duration_seconds`, which has been deprecated in Terraform AWS Provider v4.0.0 and support will be removed in a future version.
```

Comment:
The `.0.` is something internal to the SDK representation, so users don't see it
```suggestion
* `assume_role.duration` - The assume role duration represented as a string e.g. `"1h"` or `"1h30s"`. Replaces `assume_role.duration_seconds`, which has been deprecated in Terraform AWS Provider v4.0.0 and support will be removed in a future version.
```